packages/react-redux-subspace/package.json

```diff
@@ -9,13 +9,16 @@
   ],
   "license": "BSD-3-Clause",
   "main": "lib/index.js",
-  "typings": "lib/index.d.ts",
+  "module": "lib/index.es.js",
+  "typings": "index.d.ts",
```

Comment:
Personally, I really liked having the type definition next to `index.js` in the `src` directory instead of the package root.  I get that we aren't copying it to the `lib` directory anymore with babel, but is there an alternative we can use? Perhaps referencing it directly from the `src` directory or copying it manually as a seperate command?

Not a deal breaker for this PR though.

Reply:
I've just omitted the need to have additional script, it's certainly doable though - would prefer leaving this issue for a followup PR, but can do that some time later as part of this one too. Just tell me which one do u prefer

Reply:
Leave it as you have it for now.  Not having it in the root is my personal preference, but not very common amongst most libraries I've seen.